ml_metadata/tools/mlmd_bench/thread_runner.cc

```diff
@@ -27,6 +27,7 @@ limitations under the License.
 #include "tensorflow/core/lib/core/errors.h"
 #include "tensorflow/core/lib/core/status.h"
 #include "tensorflow/core/lib/core/threadpool.h"
+#include "tensorflow/core/platform/env.h"
```

Comment:
drop? is this used?

Reply:
Yes, this is used for `tensorflow::Env::Default()`.